src/AbstractFixer.php

```diff
@@ -26,6 +30,11 @@
 abstract class AbstractFixer implements FixerInterface, DefinedFixerInterface
 {
     /**
+     * @var array
```

Comment:
It can be `null` but not `Options`, the result of `OptionsResolver::resolve()` is an array which can contain any kind of value. I can use `array<string, mixed>|null` here if you want.